pkg/collector/process_test.go

```diff
@@ -360,7 +356,11 @@ func TestUDPCollectingProcess_TemplateExpire(t *testing.T) {
 		cp.Stop()
 	}()
 	cp.Start()
-	assert.NotNil(t, cp.templatesMap[1][256], "Template should be stored in the template map.")
+	template, err := cp.getTemplate(1, 256)
+	assert.NotNil(t, template, "Template should be stored in the template map.")
+	assert.Nil(t, err, "Template should be stored in the template map.")
 	time.Sleep(10 * time.Second)
```

Comment:
do we need that much sleep? Collector unit tests are taking more time to run. Just want to make sure that delay is justified and a must required. If not, we can decrease the run time of tests.

Reply:
Makes sense. I will only keep the necessary sleep (e.g. wait for collector ready, wait for template expiring). Thanks!